### PR TITLE
[GEOS] build version 3.9.0

### PR DESCRIPTION
**Pull request by visr**

### CODE REVIEW

**Comment by visr** [comment]
This is failing on PowerPC, which worked on previous versions. In [the log](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=8049&view=logs&j=1718c072-cbe0-54dd-b449-bfefb1944d59&t=995b7f31-e991-5fa8-6fc8-854e96c21f5f&l=4975) I don't see an error, just that the audit cannot find the shared library:

```
[ Info: Could not locate libgeos_c inside ["/agent/_work/1/s/G/GEOS/build/powerpc64le-linux-gnu-cxx11/Z5yb403t/destdir/lib64", "/agent/_work/1/s/G/GEOS/build/powerpc64le-linux-gnu-cxx11/Z5yb403t/destdir/lib"]
┌ Error: Built GEOS but libgeos still unsatisfied:
└ @ BinaryBuilder /depot/packages/BinaryBuilder/57NRU/src/AutoBuild.jl:761
##[error]Built GEOS but libgeos still unsatisfied:
[ Info: Could not locate libgeos, libgeos-3-9 inside ["/agent/_work/1/s/G/GEOS/build/powerpc64le-linux-gnu-cxx11/Z5yb403t/destdir/lib64", "/agent/_work/1/s/G/GEOS/build/powerpc64le-linux-gnu-cxx11/Z5yb403t/destdir/lib"]
┌ Error: Built GEOS but libgeos_cpp still unsatisfied:
└ @ BinaryBuilder /depot/packages/BinaryBuilder/57NRU/src/AutoBuild.jl:761
##[error]Built GEOS but libgeos_cpp still unsatisfied:
```

**Comment by giordano** [comment]
https://docs.binarybuilder.org/dev/troubleshooting/#Shared-library-not-built

File: G/GEOS/build_tarballs.jl [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
 
 name = "GEOS"
-version = v"3.8.1"
+version = v"3.9.0"
 
 # Collection of sources required to build GEOS
 sources = [
     "http://download.osgeo.org/geos/geos-$version.tar.bz2" =>
-    "4258af4308deb9dbb5047379026b4cd9838513627cb943a44e16c40e42ae17f7",
+    "bd8082cf12f45f27630193c78bdb5a3cba847b81e72b20268356c2a4fc065269",
```
Comment (giordano) [inline G/GEOS/build_tarballs.jl:11]: Mind using `ArchiveSource`?

Comment (visr) [reply G/GEOS/build_tarballs.jl]: Ah yeah, it's showing its age :)
Done.

**Pull request merged**